pkg/combustion/kubernetes.go

```diff
@@ -57,6 +58,25 @@ func configureKubernetes(ctx *image.Context) ([]string, error) {
 		return nil, fmt.Errorf("configuring kubernetes components: %w", err)
 	}
 
+	manifestURLs := ctx.ImageDefinition.Kubernetes.Manifests.URLs
+	localManifestsComponentDir := filepath.Join("kubernetes", "manifests")
```

Comment:
There's already a `k8sDir` constant so let's reuse it and add one for `manifests`.